src/library/util.js

```diff
@@ -134,6 +144,31 @@ export const sleep = async s => new Promise(r => setTimeout(() => { r(); }, s));
  * @returns {Boolean}
  */
 // eslint-disable-next-line max-len
-export const itemExists = (obj, param) => typeof obj === 'object' && obj !== null ? Object.prototype.hasOwnProperty.call(
-  obj, param,
-) : false;
+export const itemExists = (obj, param) =>
+  typeof obj === 'object' && obj !== null ? Object.prototype.hasOwnProperty.call(obj, param) : false;
+
+/**
+ * Checks to see if there is a sharedMicroApplication data store available for the Account or User objects
+ * @param {*} sharedMicroApplicationData the object as is, from the Account or User entities
+ * -- value each individual microApplication data store
+ * @param {*} service the service name (serverless name) of the current processing service
+ * @returns the destructured sharedMicroApplicationData
+ * @abstract the "microApplicationsToShareWith" array is a required in order to share this data set with other Micro Applications.
+ * If this field is omitted this function assumes the value is meant to be private.
+ * If the array does exist and there is a "*" in the array, then the value should be shared with all other micro applications
+ */
+export const evaluateSharedMicroApplicationData = (sharedMicroApplicationData, service) => {
+  const readableSharedMicroApplicationData = {};
+
+  // eslint-disable-next-line
+  for (const [key, value] of Object.entries(sharedMicroApplicationData)) {
```

Comment:
Is there a non eslint-angering way to do this?
Would this work:
```
Object.entries(sharedMicroApplicationData).forEach(([key, value]) => { /* loop body */ })
```

I honestly have trouble reading for... of loops these days

Reply:
Unfortunately most uses of `forEach` breaks eslint due to the "arrow functions must have a return statement". However that's a configuration we can control, would want to get input from the rest of engineering on this. But I'm okay with forEach.